app/buildpacks/node_buildpack.rb

```diff
@@ -5,6 +5,8 @@ class NodeBuildpack
   SUPPORTED = %w(5 6 8 10).freeze
   TEMPLATE  = [
     'FROM <%= base_images_registry_url %>/node:<%= node_version %>',
+    'ENV DATADOG_LOGS_SOURCE nodejs',
```

Comment:
Would it be possible to change this from `nodejs` to `node` for consistency with the Buildpack ID? Or if `nodejs` is very intentional and necessary here (maybe differentiating it from a `node` as in NodeType or whatever), let me know.

Reply:
No. It needs do follow DataDog's integration naming.

Reply:
Note that the engine that's used to run Node.js apps is also called `'node'` as seen in the [current documentation for `package.json`](https://docs.npmjs.com/files/package.json#engines):

> You can specify the version of node that your stuff works on:
> `{ "engines" : { "node" : ">=0.10.3 <0.12" } }`